### DISCUSSION TITLE
pfsense sflow configuration

### DISCUSSION THREAD

**Discussion by zveroboy152** [akvorado/akvorado#1525] [Q&A]
### Disclaimer

- [X] I have read and understood the disclaimer.

### Akvorado version

v1.11.2-56

### Bug description

Hello!

We're deploying this to get traffic details from our pfsense appliances.  However, we're having trouble getting data into this platform.

We have a basic docker install of akvorado, and a basic softflowd service configuration in pfsense (latest release).

With the below configuration we are not receiving any netflow data into akvorado.  We confirmed basic connectivity between the two platforms and traffic can pass to and from pfsense and akvorado.

Here's our configuration on the pfsense side:

Host
10.0.0.121

Specify the host to which datagrams will be sent.
Port
2055

Enter the port to which datagrams will be sent.
Sample
10

Max Flows
Specify the maximum number of flows to concurrently track before older flows are expired. Default: 8192.

Netflow version
9

Flow Tracking Level
MAC Address

Flow Timestamp Precision
Seconds

**Comment by vincentbernat** [comment]
2055 is for Netflow. For Sflow, you need to use 6343.

**Reply by vincentbernat** [reply]
It seems softflowd is exporting Netflow, so the port is correct. Follow the steps in the troubleshooting guide to pinpoint where the problem could be.
